### PR TITLE
Added more WRF example

### PR DESCRIPTION
**Pull request by xpillons**
- upgraded versions in spack builds
- fix pipeline
- added example with real life data

### CODE REVIEW

File: apps/spack/build_spack.sh [hunk 1]
```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
+set -e
 APPS_SPACK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 APP_NAME=spack
 APP_VERSION=0.16.0
 SHARED_APP=${SHARED_APP:-/apps}
-INTEL_MPI_VERSION=${INTEL_MPI_VERSION:-2020.2.254}
+INTEL_MPI_VERSION=${INTEL_MPI_VERSION:-2018.4.274} # This should be retrieved dynamically
```
Comment (garvct) [inline apps/spack/build_spack.sh:7]: Why change the Intel MPI library to an older version?

File: apps/wrf/example/namelist.tpl.input [hunk 2]
```diff
@@ -0,0 +1,94 @@
+&time_control
```
Comment (garvct) [inline apps/wrf/example/namelist.tpl.input:1]: Is this example input file useful? It alone cannot be used to run a job?

File: apps/wrf/example/namelist.tpl.wps [hunk 3]
```diff
@@ -0,0 +1,39 @@
+&share
```
Comment (garvct) [inline apps/wrf/example/namelist.tpl.wps:1]: Is this example file useful? It alone cannot be used to generate a WRF input file?

**Review by garvct — COMMENTED** [review]
Are all the WRF examples documented?

File: apps/spack/build_spack.sh (hunk 1 above)
Comment (xpillons) [reply apps/spack/build_spack.sh:7]: because 2020 is no longer in the latest images !!!

File: apps/wrf/example/namelist.tpl.wps (hunk 3 above)
Comment (xpillons) [reply apps/wrf/example/namelist.tpl.wps:1]: the namelist.wps file is used by the preparation tools and is needed to create the WRF input

**Comment by xpillons** [comment]
example documenation is still WIP

**Review by garvct — APPROVED** [review]

**Pull request merged**